### DISCUSSION TITLE
What should I do if the inventory is maliciously reported?

### DISCUSSION THREAD

**Discussion by kongxia114** [community/community#30530] [Other Feature Feedback, Questions, & Ideas]
as picture content：
![EQ}8X7)CRJFA)~)DMAVB1QL](https://user-images.githubusercontent.com/102587532/186153159-d24eee44-37d3-40b3-97d9-8526a531ab9b.png)

translate：
Someone pretended to be the author of PCL2, claimed that the PCL2 open source code on GitHub was uploaded without the author's permission, so asked GitHub to take down the code of PCL2 and asked to ban my GitHub account...
Although my account was not blocked, the PCL2 repository was blocked. I'm already contacting GitHub, hoping to get a fix..

more content：

![53FR_MDBFW_OB%~ULC~7XPH](https://user-images.githubusercontent.com/102587532/186154013-8aa6e7b5-1ec5-428b-88ed-45f154759827.png)

Preliminary verification is a DMCA report submitted by a Minecraft cheating client maker named "VeibaeMCC". This person's persecution of PCL2 is not the first time. There have been a lot of issues in the PCL2 warehouse before, because PCL2 is not good for cheating software. The "0 tolerance" caused him to be angry and carry out a series of operations on PCL2.
.

**Comment by queenofcorgis** [comment]
Hi there! The best place to go for answers to account-specific questions like this one is by opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
